src/components/DrawerMenu/DrawerMenu.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Toggle from '../Toggle';
+import ChevronDown from '../../svgs/ChevronDown';
+import ChevronRight from '../../svgs/ChevronRight';
+import { Link } from 'react-router-dom';
+import { ListItem, DrawerMenuParent, DrawerMenuChild } from './styled';
+
+const DrawerMenu = props => {
+  const { children, id, to, title, nest } = props;
+  return (
+    <Toggle>
+      {({ show, toggle }) => (
+        <DrawerMenuParent>
+          <ListItem key={id} nest={nest} onClick={toggle}>
```

Comment:
Looks like it's just a matter of moving this `onClick={toggle}` from the `ListItem` to the `span` below